app/models/opentofu_worker.rb

```diff
@@ -74,10 +74,32 @@ def unit_environment_variables
 
   def configure_service_worker_deployment(definition)
     super
-
+    # overwriting container port to be same as opentofu-runner service port i.e. in this case 6000
     definition[:spec][:template][:spec][:containers].first[:ports] = [{:containerPort => container_port}]
 
+    # ovewriting home directory to terraform home dir
+    env_var_array = definition[:spec][:template][:spec][:containers][0][:env]
+    var_name = "HOME"
+    updated_value = "/home/node"
+    env_var_array.each do |env_var_key_val|
+      if env_var_key_val[:name] == var_name
+        env_var_key_val[:value] = updated_value
+        break
+      end
+    end
+
+    # setting up runasuser. specifying '_' to avoid lint issue
```

Comment:
> `specifying '_' to avoid lint issue` 

I don't think we need this comment, this is a very common pattern in ManageIQ (and ruby generally)